app/main.py

```diff
@@ -2,0 +1,17 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file = parts[1]
```

Comment:
Ensure consistent use of double quotes for strings as per the checklist recommendation. Here, `parts[1]` and `parts[2]` are correctly used, but it's good to maintain consistency throughout the code.